packages/manager/apps/dedicated/client/app/billing/autoRenew/agreements/details/user-agreements-details.controller.js

```diff
@@ -1,22 +1,23 @@
 import filter from 'lodash/filter';
 import get from 'lodash/get';
+import toInteger from 'lodash/toInteger';
```

Comment:
Not used so it could be removed I think.
```suggestion
```

Reply:
This is removed :)